configs/config.prod.yml

```
@@ -0,0 +1,131 @@
+  notifications: true
+  webhooks: true
+
+# External APIs
+external_apis:
+  coinmarketcap:
+    api_key: ${COINMARKETCAP_API_KEY}
+    base_url: https://pro-api.coinmarketcap.com
+    timeout: 10s
+  coingecko:
+    base_url: https://api.coingecko.com/api/v3
+    timeout: 10s
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)